src/YoloDev.Expecto.TestSdk/YoloDev.Expecto.TestSdk.fsproj

```diff
@@ -20,8 +20,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="build/netcoreapp2.1/YoloDev.Expecto.TestSdk.props" Pack="true" PackagePath="build\netcoreapp2.1\" />
-    <None Include="build/netcoreapp2.1/_._" Pack="true" PackagePath="lib\netcoreapp2.1\" Visible="false" />
+    <None Include="build/netcoreapp3.1/YoloDev.Expecto.TestSdk.props" Pack="true" PackagePath="build\netcoreapp3.1\" />
+    <None Include="build/netcoreapp3.1/_._" Pack="true" PackagePath="lib\netcoreapp3.1\" Visible="false" />
     <None Include="build/net5.0/_._" Pack="true" PackagePath="lib\net5.0\" Visible="false" />
```

Comment:
Shoud it be net6.0, @xperiandri ?

Seems like this is the reason of CI failures

Reply:
@Alxandr why `YoloDev.Expecto.TestSdk.props` went only to `netcoreapp2.1` folder? Isn't it needed for each .net version?

Reply:
I don't remember to bra frank. But what I do know is that if you only target `netcoreapp2.1`, that one will get used by newer ones.

Reply:
OK, let's try as it is now